### DISCUSSION TITLE
Vercel-Github Expert

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#32407] [Help]
I paid a development company to redesign my e-commerce site. Now, they have access to all my files and API and I don't. It's on vercel. I have ownership. I need some help getting my account access and also someone that can help with some design changes an issue I need to be fixed. And possibly a lot more work from there.

**Comment by 1finedev** [comment]
i am available. Kindly email me on emmanuel.bayode.0388@gmail.com
Cheers
